### ISSUE TITLE
Homepage Redesign 

### ISSUE THREAD

**Issue by kristine-eudey** [hackforla/website#2057]
### Overview
We need the new homepage design to be completed so that it can move to development

### Action Items

- [x] Bonnie and Kristine to have a working session to pick up from previous design work and come up with a design solution that will be moved forward to development. 
- [x] Low Fidelity Design 
- [x] Mid Fidelity Design
- [x] High Fidelity Design 
- [x] Final content in place:
   - [x] Copy
   - [x] Images
 - [x] Audit of final design to ensure that Design System requirements are met

### Resources/Instructions
[Figma - Homepage Redesign](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=13023%3A90640
)

**Comment by kristine-eudey** [comment]
I will complete low-fidelity design by Thursday 8/5, at which point it will be reviewed with the rest of design team and further issues will be created to delegate additional design work.

**Comment by kristine-eudey** [comment]
@ExperimentsInHonesty please take a look at the updated design here: [Figma - Homepage Redesign 8/5/21](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=13558%3A91414), and let me know your thoughts. 
The UI needs further work before the low-fi is ready to head to developers to ensure the general structure/layout is as close to design system requirements (font sizing, margins, buttons, etc.) as possible before handing off.

**Comment by kristine-eudey** [comment]
Please see updated low/mid fidelity homepage design [here](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=13774%3A92283), with changes made as per our discussion on 8/8.

**Comment by kristine-eudey** [comment]
Most recent update to homepage redesign here: [Homepage Redesign 8/15/21](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=14020%3A92817) Most current design is in pink rectangle.

**Comment by kristine-eudey** [comment]
@ExperimentsInHonesty Most recent version here, in pink rectangle: [Homepage Redesign - 8/22/21](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=14287%3A93030)

**Comment by kristine-eudey** [comment]
All recently discussed edits have been made and, pending approval, I think the MVP v2 is ready to go. I will work on MVP v1 in the coming days, which is the version with a simplified Program Areas section while we wait for project impact content to be finalized. 

See most up to date version here:[Figma - Homepage Redesign: 9/3/2021](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=15145%3A98212)

**Edits made to this version:**
- Top 10 used technologies icons are now featured. See Issue --> #2130 
- CoP icons are finalized (pending any desired changes by dev team to their icon, will ask at Sunday meeting)
- This change was not discussed, but I made a design edit to the "We use technology to make an impact" section. This section kept feeling off to me, was way too much text without enough visual impact to keep a person there and lead them through it. Got feedback from designers on the team that the "join us" and "about us" sections looked like two different, yet equal, options to choose from rather than as leading to distinct content. I changed the flow of content in order to bring in a more linear rhythm, leading reader from one section to the next vertically. I also brought in a background graphic to help break up the large amount of text all on a white background. 
- All spacing and fonts have been checked against design system conventions, and use design system components where possible.

**Comment by kristine-eudey** [comment]
View most up to date versions **v.1 - abbreviated program areas section**  & **v.2 - full program area section** here: [Figma - Homepage Redesign](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=15505%3A99314)

**Comment by kristine-eudey** [comment]
Completed [homepage figma design ](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=16091%3A101199)

**Notes:**
- Section heading font size has been changed from Title 1 to Title 2. I will discuss this with design team at tonight's meeting and decide which version to put forward, both versions are fully designed and labeled on the Figma. My preference is for the updated version with Title 2 section headings. Barring this potential change, the design is complete as has been approved by Bonnie/Product team. 
- I've designed an alternate hero section, which would replace the current hero image and reformat the text. For now, the version with the old hero image should be built and we will test both versions with users. Bonnie will also speak to people linked to the organization to gauge the effect changing this image away from the old one might have. I will create an issue to track progress on the hero image question, and will update this with link to the issue when that's been done.
